src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/avenir-book.ttf");
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: italic;
+  font-weight: normal;
+}
+
+body {
+  box-sizing: border-box;
+}
+
+.input-search {
+  width: 100%;
+  outline: none;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  border: 1px solid #f0f2f5;
+  box-shadow: 0 1px 8px rgba(58, 69, 82, 0.1);
+  border-radius: 4px;
+  color: #435c79;
+}
+
+.input-search:hover {
+  box-shadow: 0 5px 10px rgba(54, 62, 71, 0.2);
+}
+
+.input-search:focus {
+  background-color: transparent;
+  border-radius: 10px;
+}
+
+.big-search {
+  margin: 20px 0;
+}
+
+.form-big-search,
+.smallSearch__form {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input-big-search {
+  padding: 23px 20px 23px 62px;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.input-small-search {
+  padding: 10px 20px 11px 33px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 19px;
+}
+
+.input-big-search:focus {
+  font-weight: 600;
+}
+
+.input-small-search:focus {
+  font-weight: 900;
+}
```

Comment:
It should be the same for both small and big searches, also, move it to the main class, no need to leave it just for focus state